src/components/admin/InfoRequestGrid.tsx

```diff
@@ -42,14 +42,23 @@ export default function InfoRequestGrid() {
 
   return (
     <DataGrid
+      style={{
+        background: 'white',
+        height: 'calc(100vh - 300px)',
+        border: 'none',
+        width: 'calc(100% - 48px)',
+        left: '24px',
```

Comment:
Is left: '24px' going to be applied if the position is not absolute?

Reply:
'left' and 'width' removed